scalalib/src/mill/scalalib/CoursierModule.scala

```diff
@@ -317,6 +318,28 @@ object CoursierModule {
 
       res.orderedDependencies
     }
+
+    def getArtifacts[T: CoursierModule.Resolvable](
```

Comment:
Thanks for this. I was working on SBOM generation and I was not sure what the best way was to expose the detailed info. This will provide all the info I need.